src/multi-locale.ts

```diff
@@ -211,23 +237,23 @@ export function generateLocalizedAssets(
 		}));
 	};
 
-	// Apply after minification since we don't want to
-	// duplicate the costs of that for each asset
 	if (isWebpack5Compilation(compilation)) {
-		// Happens after PROCESS_ASSETS_STAGE_OPTIMIZE_SIZE
+		const Comp = (compilation.constructor as typeof WP5.Compilation);
+		const stage = runBeforeMinification
+			? Comp.PROCESS_ASSETS_STAGE_DERIVED
```

Comment:
As discussed in #21, introducing assets pre-minification will incur a very large performance cost and negate a major optimization this plugin offers.

Reply:
Agreed. See comment https://github.com/privatenumber/webpack-localize-assets-plugin/pull/26/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R71